doc/guides/collaborator-guide.md

```diff
@@ -210,20 +210,30 @@ All fixes must have a test case which demonstrates the defect. The test should
 fail before the change, and pass after the change.
 
 All pull requests must pass continuous integration tests. Code changes must pass
-on [project CI server](https://ci.nodejs.org/). Pull requests that only change
+on [project CI server](). Pull requests that only change
 documentation and comments can use GitHub Actions results.
 
 Do not land any pull requests without a passing (green or yellow) CI run.
-For documentation-only changes, GitHub Actions CI is sufficient.
-For all other code changes, Jenkins CI must pass as well. If there are
-Jenkins CI failures unrelated to the change in the pull request, try "Resume
-Build". It is in the left navigation of the relevant `node-test-pull-request`
-job. It will preserve all the green results from the current job but re-run
-everything else. Start a fresh CI if more than seven days have elapsed since
-the original failing CI as the compiled binaries for the Windows and ARM
-platforms are only kept for seven days.
-
-#### Useful CI jobs
+A green GitHub Actions CI result is required. A passing
```

Comment:
Yellow is also OK.

Reply:
Oh, wait, GitHub Actions CI. Yeah, so maybe it is just green.

We've never required GitHub Actions CI be green before. This would be a big change in the process. I'm not opposed, necessarily, but I want to highlight this here so it doesn't escape notice.